modules/ROOT/pages/dw-operators.adoc

```diff
@@ -363,3 +363,359 @@ xref:dataweave-variables.adoc#example_local_variable[Examples: Local DataWeave V
 | `if else` a| An `if` operator evaluates a conditional expression and returns the value under the `if` only if the conditional expression is true. Otherwise, it returns the expression under `else`. Every `if` expression must have a matching `else` expression. See xref:dataweave-flow-control.adoc#control_flow_if_else[if else] for an example.
 | `else if` | An `else` operator chains expressions together within an if-else construct by incorporating `else if`. See xref:dataweave-flow-control.adoc#control_flow_else_if[else if] for an example.
 |===
+
+== Update Operator
+
+DataWeave 2.0 supports the update operator that enables you to update a given field on a data structure with a new value. This use case is very common and was not simple to do in DataWeave.
+
+_Introduced in DataWeave 2.3.0. Supported by Mule 4.3 and later._ +
+
+Example: Using Update Operator, let's say that you have an object:
```

Comment:
Suggest following the format of the examples in other sections on this page or picking a single format for them to follow.

Reply:
Done.